src/cephadm/cephadm.py

```diff
@@ -4692,6 +4693,35 @@ def command_list_images(ctx: CephadmContext) -> None:
     # print default images
     cp_obj.write(sys.stdout)
 
+
+def update_service_for_daemon(ctx: CephadmContext, daemon_list: list) -> None:
+    """ Update the unit.meta file of daemon with required service name for valid daemons"""
+
+    data = {'service_name': ctx.service_name}
+    update_daemon = ctx.name.split(',')
+    available_daemons = [d['name'] for d in daemon_list]
+    # check if all the daemon names are invalid
+    if not list(set(update_daemon) & set(available_daemons)):
+        raise Error(f'Error EINVAL: Invalid daemon name {update_daemon}')
+    for name in update_daemon:
+        if name not in available_daemons:
+            print(f'Daemon {name} does not exists')
+            continue
+        path = os.path.join(ctx.data_dir, ctx.fsid, name, 'unit.meta')
+        update_meta_file(path, data)
+
+
+@infer_fsid
+def command_update_service(ctx: CephadmContext) -> int:
+    """update service for provided daemon"""
+    daemon_list = list_daemons(ctx, detail=False)
+    if not len(daemon_list):
```

Comment:
List in python can be treated as booleans as well:

```suggestion
    if not daemon_list:
```

Reply:
Yes